packages/react/src/Icon/Icon.module.css

```diff
@@ -1,121 +1,95 @@
-.Icon--badge {
-  width: var(--base-size-44);
-  height: var(--base-size-44);
-  border-radius: 50%;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+.Icon {
+  box-sizing: content-box;
+  border-radius: 9999px;
   padding: var(--base-size-12);
 }
 
-.Icon--color-default {
-  color: var(--brand-Icon-color-default);
+.Icon--noBackground {
+  background-color: initial !important;
```

Comment:
Cool with this `!important`?

Reply:
It looks like we need this because we currently apply a background color by default to all Icons, even when background is not required or `false`, is that correct?

If so, this feels a little inverted, because we shouldn't apply that style unless we need it. In an edge case it could also cause a specificity issue if someone tried to override the background color, though they could workaround that too.

If we did invert it, presumably we'd need to add a `with-background-color` class for each of the colors? In terms of lesser evils, keeping the `!important` might be okay. Any other ways to solve this @joshfarrant?

Reply:
Just noticed we used to have a separate rule for badges

`.Icon--color-{hue}.Icon--badge`

I think that's easier to reason about tbh. What was the motivation behind removing that?

Reply:
Yea I'm happy to revert that change. Keeping both the `color` and `background-color` together made more sense to me and cut down on the amount of CSS in the file, but I don't have strong opinions so I'm happy to revert it